.gitignore

```diff
@@ -4,14 +4,15 @@
 /node_modules
 /.pnp
 .pnp.js
-
+package-lock.json
 # testing
 /coverage
 
 # production
 /build
 
 # misc
+.idea/
```

Comment:
This is the sort of thing that should be on your [personal gitignore](https://medium.com/@peter_graham/how-to-create-a-local-gitignore-1b19f083492b)